python/sglang/srt/models/deepseek_v2.py

```diff
@@ -2653,3 +2728,49 @@ class DeepseekV3ForCausalLM(DeepseekV2ForCausalLM):
 
 
 EntryClass = [DeepseekV2ForCausalLM, DeepseekV3ForCausalLM]
+
+# Add environment variable for debugging: limit model to only load up to and including first MoE block
```

Comment:
btw wondering whether `--json-model-override-args '{"num_hidden_layers": 5}'` helps for debug

Reply:
didnt know about this option. maybe it does, Ill try next time, I just limited up to and including first MoE block manually). I'll ofc reverse that change.